book/src/pod-inline-volume.md

```diff
@@ -0,0 +1,53 @@
+# Pod Inline Volume Support
```

Comment:
Can this replace "ephemeral local volumes" page instead?

Reply:
I replaced the link, take a look to see if that works for your suggestion.

Reply:
Looks good can you rename the file to match the title of the page?

Reply:
@msau42 Renamed.